runtime/crab/src/lib.rs

```diff
@@ -691,54 +691,30 @@ impl dvm_rpc_runtime_api::ConvertTransaction<OpaqueExtrinsic> for TransactionCon
 	}
 }
 
+fn migrate() -> Weight {
+	// --- paritytech ---
+	#[allow(unused)]
+	use frame_support::migration;
+
+	migration::remove_storage_prefix(b"CrabIssuing", b"TotalMappedRing", &[]);
+
+	// TODO: Move to S2S
+	// const CrabIssuingPalletId: PalletId = PalletId(*b"da/crais");
```

Comment:
Todo when s2s bridge is ready and RING's mapping token contract is created:

https://github.com/darwinia-network/darwinia/issues/726